src/backend/aspen/workflows/ingest_gisaid/ingest.sh

```diff
@@ -36,7 +36,7 @@ fi
 end_time=$(date +%s)
 
 # create the objects
-entity_id=$(/aspen/.venv/bin/python /aspen/src/backend/workflows/ingest_gisaid/save.py \
+entity_id=$(/aspen/.venv/bin/python /aspen/src/backend/aspen/workflows/ingest_gisaid/save.py \
                                     --aspen-workflow-rev "${aspen_workflow_rev}"  \
                                     --aspen-creation-rev "${aspen_creation_rev}"  \
                                     --start-time "${start_time}"                  \
```

Comment:
nitpicky: can you fix the `\`

Reply:
wait what's wrong with it?? 😂

Reply:
once upon a time, all the backslashes lined up.